tests/introspection/ApiSignatureTest.cs

```diff
@@ -181,7 +181,18 @@ protected virtual bool Skip (Type type)
 
 		protected virtual bool Skip (Type type, MethodBase method, string selector)
 		{
+			if (method is MethodInfo mf) {
+				if (IsNullableType (mf.ReturnType))
+					return true;
```

Comment:
We should not skip
Else the other check won't be done and we'll miss errors 
Instead we could detect nullable and see the the return or parameters are something bindas support